uberon_edit.obo

```diff
@@ -195254,12 +195254,13 @@ id: UBERON:8410010
 name: fimbria of uterine tube
 def: "A fringe of tissue around the ostium of the fallopian tube, in the direction of the ovary." [WikipediaVersioned:Fallopian_tube&oldid=998395727]
 synonym: "fimbria of fallopian tube" EXACT []
-synonym: "fimbriae of fallopian tube" EXACT []
-synonym: "fimbriae of uterine tube" EXACT []
+synonym: "fimbriae of fallopian tube" EXACT [] {http://purl.obolibrary.org/obo/uberon/core#PLURAL=""}
+synonym: "fimbriae of uterine tube" EXACT [] {http://purl.obolibrary.org/obo/uberon/core#PLURAL=""}
```

Comment:
Looks like some error in how this was added.

Should be like this.
![image](https://user-images.githubusercontent.com/112839/107023912-331e0b00-679f-11eb-8580-7de0c4e352a5.png)